runtime/queries/nginx/highlights.scm

```diff
@@ -0,0 +1,48 @@
+(comment) @comment
+
+(block_directive
+	(directive) @type)
+
+[
+	"{"
+	"}"
+	"("
+	")"
+	"["
+	"]"
+] @punctuation.bracket
+
+(simple_directive
+	(directive) @function)
+
+[
+	";"
+] @punctuation.delimiter
+
+((generic) @keyword
+ (#any-of? @keyword
+ 	"on"
+ 	"off"
+ 	"any"
+ 	"auto"))
+
+(modifier) @operator
+
+(generic) @variable
+
+(string) @string
+
+(number) @constant.numeric
+(metric) @constant.numeric
+
+(variable) @variable.parameter
+
+(regex) @string
+
+(modifier) @keyword.operator
+
+(lua_block_directive
+	"access_by_lua_block" @function)
+
+; Lua Debug
+(lua_code) @definition.type
```

Comment:
this pattern can probably be removed: `definition.type` is not in our captures (https://docs.helix-editor.com/master/themes.html#syntax-highlighting) so it shouldn't be themed anyways. The `lua_code` node is used later in `injections.scm` so it should be highlighted as lua instead

Reply:
I removed it